config/checkstyle-checks.xml

```diff
@@ -39,11 +39,11 @@
   <!-- Tone down the checking for test code -->
   <module name="SuppressionSingleFilter">
     <property name="checks" value="JavadocPackage"/>
-    <property name="files" value=".*[\\/]src[\\/](test|it)[\\/]"/>
+    <property name="files" value=".*[\\/]src[\\/](test|it|xdocs-test)[\\/]"/>
```

Comment:
It seems to me this makes more sense to be named `xdocs-it` or `xdocs-examples`. This area is not really about tests.

Reply:
`xdocs-examples` makes sense. I have always wondered, what is the relevance of the `it` term?

Reply:
I am fine with `xdocs-examples`.

@romani Thoughts?

Reply:
I am ok to rename, as we finally agreed that such "tests" aren't part of coverage, so they are not really critical testing. And we end up in name Example, and structure of them will be a bit unusual. So they similar, but not tests.

So , yes, it is `xdoc-examples`

Reply:
Done. Renamed to `xdocs-examples`